other/underworld/undead

```diff
@@ -1,8 +1,7 @@
 **Undead** | Solo Miscellaneous - Underworld Team
 __Basics__
-The Undead are the vampire’s secret army. If a demonized player were to die during the night, they will instead lose their original role and become an Undead.
+The Undead is a dead player who pretends to be alive, and they are part of the vampire’s secret army. If a demonized player dies during the night, they will not die, and will instead lose their original role and become an Undead.
```

Comment:
phrasing here makes it seem like the player is actually a dead participant, and not just a participant who had their role changed to Undead by getting attacked while demonized

Reply:
this is actually just from the original description (just moved to basics instead of details), however we can change it.